ui/app/styles/components/codemirror.scss

```diff
@@ -13,6 +13,12 @@ $dark-bright: lighten($dark, 15%);
   min-height: 500px;
 }
 
+.namespace-editor {
+  .CodeMirror-scroll {
+    min-height: 170px;
```

Comment:
Default for this is 500px; did it feel too large? Policies are typically longer in config than namespaces, so is the 170px argument just that they need less space?

Reply:
Yeah, just picked a smaller number based on what felt good. - Any gut reaction while you were using this? If not, then I think that's a good sign?

Reply:
It seemed small to me but I don't write namespaces very often so don't take my word for it